easybuild/easyconfigs/g/GLib/GLib-2.60.1-GCCcore-8.2.0.eb

```diff
@@ -34,7 +42,7 @@ preconfigopts = "export PYTHON=python && "
 
 configopts = "--buildtype=release --default-library=both "
 
-postinstallcmds = ["sed -i -e 's|#!.*python|#!/usr/bin/env python|' %(installdir)s/bin/*"]
+postinstallcmds = ["sed -i -e 's|#!.*python[0-9.]*$|#!/usr/bin/env python|' %(installdir)s/bin/*"]
```

Comment:
Another good use case for https://github.com/easybuilders/easybuild-framework/issues/2869

Doesn't need to block this PR, but we should change this easyconfig once we have global support for `fix_python_shebang_for`...